src/gozilla/storage/database/db.go

```diff
@@ -0,0 +1,482 @@
+package database
+
+import (
+	"fmt"
+	"time"
+
+	"gorm.io/gorm"
+	setting "aaditya-singh/gozilla/src/storage/config"
+)
+
+const (
+	checkSchedulerTables = `
+		SELECT column_name FROM information_schema.columns
+		WHERE table_catalog = '%s'

```

Comment:
any specific reason to use, QUERYs?
GORM has the inbuilt query functionality right?
Is it because of the free hand to configure queries?

Reply:
Hi @Kaushl2208 , not much! I've been experimenting with things. Initially, I wasn't familiar with `PostgreSQL`, so I was just trying to make things fit. Now that I've learned more about it, I plan to incorporate `functions`, `procedures`, and make use of `JSONB` and `transactions` for better updates and retrieval in table. Since our `job` table is linked to several other tables, operations on it will also impact those tables.
In the meantime, I'll be using `gorm: crud interface & query` part to handle queries.